addons/l10n_in_pos/static/src/compnay_state_dialog/compnay_state_dialog.js

```diff
@@ -0,0 +1,25 @@
+/** @odoo-module */
+
+import { Dialog } from "@web/core/dialog/dialog";
+import { Component } from "@odoo/owl";
+import { usePos } from "@point_of_sale/app/store/pos_hook";
+
+export class companyStateDialog extends Component {
+    static components = { Dialog };
+    static template = "l10n_in_pos.companyStateDialog";
+    static props = {
+        close: Function,
+    };
+
+    setup() {
+        this.pos = usePos();
+    }
+
+    redirect() {
+        window.location = "/odoo/settings/4/action-52/" + this.pos.company.id;
```

Comment:
I don't think the action identifier will be the same in all DBs.

Reply:
I have updated the code.